SingularityExecutor/src/main/java/com/hubspot/singularity/executor/task/SingularityExecutorTaskProcessCallable.java

```diff
@@ -115,4 +131,93 @@ private boolean runHealthcheck(Process process) {
       return false;
     }
   }
+
+  private boolean runHealthcheckV2(Process process, HealthcheckOptions maybeOptions) {
+    String taskAppDirectory = task.getTaskDefinition().getTaskAppDirectory();
```

Comment:
just confirming, does this equate to essentially `./healthcheck-config.json` from within task code?

Reply:
Yea exactly.